code/game/objects/items/ego_weapons/he.dm

```diff
@@ -91,6 +92,25 @@
 	attribute_requirements = list(
 							FORTITUDE_ATTRIBUTE = 40
 							)
+	var/living = FALSE
```

Comment:
Why is it here at all?

Reply:
So that it only works ON kill, it checks if it's alive before and after your attack, and if they're alive before your attack and dead after then you get the boost

Reply:
It does not have to be a **weapon** variable.
It can be easily done in the attack proc itself.